EIC-EA/EIC-community-D8
# EICNET-906: Fixed labels for media fields, added topics and region fields

Tests:

- Create a group with a wiki
- Log in as a trusted user
- **You should see the labels of the media fields**
- **You should see the Topics and Regions fields**

Questions:

- Should the body text type be visible
- Do we need the media fields?
- Should the 'send message to activity stream' checkbox be hidden
- Should the wiki menu be hidden
- should the log revision field be hidden
- What type of fields are the themes and tags field that need to be added
